ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/config/TestAuditYamlConfigurationLoader.java

```diff
@@ -86,6 +86,7 @@ public void testEmptyWhitelistIsStillValid()
         AuditConfig config = givenLoadedConfig(properties);
 
         assertThat(config.getYamlWhitelist()).isEmpty();
+        assertThat(config.isPostLogging()).isFalse();
```

Comment:
This test should probably sit on the tc above (or both)

Reply:
true, fits better there